index_alias_impl.go

```diff
@@ -44,6 +44,13 @@ func NewIndexAlias(indexes ...Index) *indexAliasImpl {
 	}
 }
 
+func (i *indexAliasImpl) Indexes() []Index {
+	i.mutex.RLock()
+	rv := i.indexes
+	i.mutex.RUnlock()
+	return rv
+}
+
```

Comment:
btw, this change seems unrelated to theme of this force-merge change?

Also, it needs more documentation?  e.g., the locking here isn't enough, as the Add/Remove() methods can mutated the underlying i.indexes?  Need to warn the user to make all their add/remove() calls upfront before calling this Indexes() implementation as isn't safe in the face of concurrent add/remove() calls?

Reply:
this change is related, this help at a cbft level for triggering force merges to remote partitions, we need to know which all partitions are hidden behind an alias. Something similar to mocking the Multisearch sort mult index functionality at the application level..

yes, let me add more commentary there..